### PR TITLE
Add meta tags and meta tags blocks

### PR DESCRIPTION
**Pull request by Abibubble**
# Business Analysis 🟠  Code Institute 🟠  Community Project - Pull Request 🎉🎉

---

## Pull Request References:
- #46 Meta tags - This PR is not complete with this PR, don't close it.

## PR contains:
- Meta tags on base.html file
- Blocks for adding content to meta tags for each individual page
- Content added to blocks on pages for meta tags related to that individual page
- Full URL still needed when we deploy
- Images for social media sharing are still required, but can only be added once the site is almost complete

## Breaking changes
- None

## Screenshots/Gifs/Media
- None

### CODE REVIEW

File: templates/create_organisation.html [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+
+{% block metas %}
+<link rel="canonical" href="/add_business">
+<meta name="description" content="Add information for the  Dún Laoghaire-Rathdown County Council to hold about a new business.">
+<meta property="og:description" content="Add information for the  Dún Laoghaire-Rathdown County Council to hold about a new business." />
+<meta name="image" property="og:image" content="LINK TO IMAGE" />
+<meta property="og:image:alt" content="A form to add information to be held by Dún Laoghaire-Rathdown County Council about this business" />
+<meta property="og:url" content="/add_business" />
+<meta name="twitter:description" content="Add information for the  Dún Laoghaire-Rathdown County Council to hold about a new business.">
+<meta name="twitter:image" content="LINK TO IMAGE">
```
Comment (auxfuse) [inline templates/create_organisation.html:14]: Do we want to create a standalone issue for this to ensure we don't miss it later on in the development process and forget to update the content value @Abibubble ?

**Review by auxfuse — CHANGES_REQUESTED** [review]
Hi @Abibubble excellent PR! And thank you for taking on this massive task of A11Y & SEO. Just a couple of things for feedback and discussion. Very open here to taking your professional knowledge and stance on them too. So don't be afraid to disagree with me here 😂😂

File: templates/create_organisation.html (hunk 1 above)
Comment (Abibubble) [reply templates/create_organisation.html:14]: My thought was to not close this issue with this PR. The issue has checkboxes for things that are done, which includes the things still left do to (like the images). Happy to split it up into multiple issues if that's preferred, not sure what best practice is on this (I'd normally do multiple issues, but didn't want to clutter the board up) :)

File: templates/create_organisation.html [hunk 2]
```diff
@@ -1,4 +1,20 @@
 {% extends "base.html" %}
+
+{% block extra_title %} Add Organisation{% endblock %}
+{% block extra_keywords %}, organisation, organisations, add, create{% endblock %}
+
+{% block metas %}
+<link rel="canonical" href="/add_business">
```
[the remark this answers is not in the document]
Comment (Abibubble) [reply templates/create_organisation.html:7]: Oh my gosh, please yes!!! I was following what others had done :) Will update this and push to the PR :)

**Comment by Abibubble** [comment]
@auxfuse Updated the indenting, just let me know what's the preference regarding the issue splitting up and will do that 😀

**Pull request merged**